posthog/models/filters/mixins/common.py

```diff
@@ -450,6 +450,16 @@ def entity_math_to_dict(self):
         return {"entity_math": self.target_entity_math} if self.target_entity_math else {}
 
 
+class EntityOrderMixin(BaseParamMixin):
```

Comment:
I'm confused, why do we need to introduce this mixin?

Is it because `order` isn't a required param?